pkg/kubelet/kubelet.go

```diff
@@ -1361,6 +1361,23 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 	}
 }
 
+// Returns Docker version for this Kubelet.
+func (kl *Kubelet) GetDockerVersion() (string, error) {
+	if kl.dockerClient == nil {
+		return "", fmt.Errorf("No Docker client")
```

Comment:
"no docker client"

errors are all lower case in Go by convention.  (don't ask me why...)